etl/data_management/DataManager.py

```diff
@@ -38,6 +38,7 @@ class DataManager:
     enhanced_data_location = os.path.join(ROOT_DIR, 'data_enhancement/data')
     backup_directory = os.path.join(ROOT_DIR, 'data_management', 'backups')
     upload_directory = os.path.join(ROOT_DIR, 'data_management', 'upload')
+    file_upload_data_origin = os.path.join(ROOT_DIR, 'data_management', 'upload_data_origin.log')
```

Comment:
Wouldn't it be better if the log file is also saved in the log directory?
`data_management/logs/upload_data_origin.log`

Reply:
done

Reply:
changed location of upload_data_origin.log to the same location of the other logs instead (`etl\logs`)